configure.ac

```diff
@@ -145,6 +146,20 @@ CPPFLAGS="-D_REENTRANT -D_THREAD_SAFE $CPPFLAGS"
 # For host type checks
 AC_CANONICAL_HOST
 
+#For a working getsockopt() optname=SO_ERROR
+dnl AC_MSG_CHECKING([for getsockopt optname SO_ERROR)])
```

Comment:
Note, please send any fixes to this change as new pull requests.

On Sat, Jan 16, 2016 at 2:17 PM, Luca Boccassi <notifications@github.com>
wrote:

> In configure.ac
> <https://github.com/zeromq/libzmq/pull/1709#discussion_r49931595>:
>
> > @@ -145,6 +146,20 @@ CPPFLAGS="-D_REENTRANT -D_THREAD_SAFE $CPPFLAGS"
> >  # For host type checks
> >  AC_CANONICAL_HOST
> >
> > +#For a working getsockopt() optname=SO_ERROR
> > +dnl AC_MSG_CHECKING([for getsockopt optname SO_ERROR)])
>
> On the Debian bug discussion [1], @sthibaul <https://github.com/sthibaul>
> mentioned that this sockopt is in fact available on Hurd. Also doesn't look
> like HAVE_SOCKLEN_T is used. Any reason to keep this section?
>
> [1] https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=799235
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zeromq/libzmq/pull/1709/files#r49931595>.
>